skel/problem/generators/generators.yaml

```diff
@@ -62,32 +62,32 @@ data:
   invalid_inputs:
     # Add invalid testcase to ensure that input_validators correctly rejects them.
     data:
-      empty:
-          in: ""
-      newline:
-          in: "\n"
-      random:
-          in: YVRtr&*teTsRjs8ZC2%kN*T63V@jJq!d
-      not_printable_1:
-          in: "\x7f"
-      not_printable_2:
-          in: "\xe2\x82\xac"
-      unicode:
-          in: "¯\\_(ツ)_/¯"
+      too_many_tokens:
+          in: 2 2 2
+      leading_whitespace:
+        in: " 2 2"
+      extra_line:
+          in: "2 2\n\n"
 
   invalid_answers:
     # Add valid testcase with invalid answers to ensure that the answer_validators correctly rejects them.
     # (The output validator is also called with these)
     data:
-      #dont_start_with_whitespace:
+      #leading_whitespace:
       #    in: "<valid testcase>"
       #    ans: " <valid answer>"
+      in: 2 2
```

Comment:
hmm not sure how I feel about hardcoding these problem specific examples into the skel. Just putting `data: {}` is also fine I'd say.


This one is missing a testcase name.